docs/security/configuration/client-auth.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: Client-based authentication
+parent: Configuration
+grand_parent: Security
+nav_order: 50
+---
+
+# Client-based authentication
+
+After obtaining your own certificates either from a certificate authority (CA) or by [generating your own certificates using OpenSSL](../generate-certificates), you can start configuring Elasticsearch to authenticate a user using a client certificate.
+
+## Enabling client authentication
+
+To enable client authentication in Elasticsearch, you need to first set `clientauth_mode` in `elasticsearch.yml` to either `OPTIONAL` or `REQUIRE`:
```

Comment:
Technically you're enabling it in the security plugin, not Elasticsearch, so we want something like:

To enable client certificate authentication, you must first set...